lms/djangoapps/grades/models.py

```diff
@@ -277,8 +277,8 @@ class Meta(object):
     usage_key = UsageKeyField(blank=False, max_length=255)
 
     # Information relating to the state of content when grade was calculated
-    subtree_edited_timestamp = models.DateTimeField('last content edit timestamp', blank=False)
-    course_version = models.CharField('guid of latest course version', blank=True, max_length=255)
+    subtree_edited_timestamp = models.DateTimeField(u'Last content edit timestamp', blank=True, null=True)
+    course_version = models.CharField(u'Guid of latest course version', blank=True, max_length=255)
```

Comment:
Will address if currently-running tests fail; otherwise we'll let it go in the interest of getting this PR merged.